enterprise/frontend/src/metabase-enterprise/moderation/components/ModerationIssueThread.jsx

```diff
@@ -79,3 +103,59 @@ export function ModerationIssueThread({
     </div>
   );
 }
+
+CommentForm.propTypes = {
+  className: PropTypes.string,
+  onSubmit: PropTypes.func.isRequired,
+  onCancel: PropTypes.func.isRequired,
+  isPending: PropTypes.bool,
+};
+
+function CommentForm({ className, onSubmit, onCancel, isPending }) {
+  const textAreaRef = useRef();
+  const [value, setValue] = useState("");
+  const isEmpty = value.trim().length === 0;
+
+  useEffect(() => {
+    const textAreaEl = textAreaRef.current;
+    if (textAreaEl) {
+      textAreaEl.focus();
+    }
+  }, []);
```

Comment:
Am I right this effect automatically focuses the textarea?
If so, should we maybe use `<textarea autofocus />`?

Reply:
Oh ya, good suggestion. Fixed